__tests__/app.test.js

```diff
@@ -42,7 +42,7 @@ describe("GET /api/topic", () => {
 });
 
 describe("GET /api/articles/:article_id", () => {
-  it("200: response with a single article object of the id", () => {
+  it("200: response with a single article object of the id, with comment count 0", () => {
```

Comment:
Just a point for the future, ideally you need to make a new test, for the new feature, not change previous ones